precompiles/authorization/events.go

```diff
@@ -121,3 +129,44 @@ func EmitRevocationEvent(args cmn.EmitEventArgs) error {
 
 	return nil
 }
+
+// EmitIBCTransferAuthorizationEvent creates a new IBC transfer authorization event emitted on a TransferAuthorization transaction.
+func EmitIBCTransferAuthorizationEvent(
+	event abi.Event,
+	ctx sdk.Context,
+	stateDB vm.StateDB,
+	precompileAddr, granteeAddr, granterAddr common.Address,
+	allocations []cmn.ICS20Allocation,
+) error {
+	topics := make([]common.Hash, 3)
+
+	// The first topic is always the signature of the event.
+	topics[0] = event.ID
+
+	var err error
+	topics[1], err = cmn.MakeTopic(granteeAddr)
+	if err != nil {
+		return err
+	}
```

Comment:
This function is flagged as having no tests?

Reply:
Not directly but it's being checked in each of the places where this event is emitted like in ics20.